doc/source/guide/introduction/installing.rst

```diff
@@ -2,36 +2,124 @@
 Installing
 ##########
 
-Installing odl is intended to be straightforward, and this guide is meant for new users. To install odl you need to do the following steps:
+Installing odl is intended to be straightforward, and this guide is meant for new users. For a minimal installation you should do the following steps:
 
 1. `Install Python`_
-2. `Install Git`_
-3. `Install ODL`_
-4. `Run tests`_
+2. `Install ODL`_
+3. `Run tests`_
 
 If you have done any step before, you can ofc skip it.
 
 Install Python
 ==============
-To begin with, you need a python distribution. If you are an experienced user, you can use any distribution you'd like.
+To begin with, you need a Python distribution. Any modern Python distribution should do for the core library, but some extensions require CPython.
```

Comment:
Perhaps simply rename the `Python 2 vs 3` header to `What python version to use` and discuss that there? Feels like it would "dominate" this part by too much text.